### PR TITLE
added expression tab for querying VC or combined studies

### PR DESCRIPTION
**Pull request by yichaoS**
Fix issue: https://github.com/cBioPortal/iViz/issues/429
Heroku demo: https://cbioportal-pr-2290.herokuapp.com/

### CODE REVIEW

File: portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp [hunk 1]
```diff
@@ -444,41 +444,7 @@ if (sessionError != null) {  %>
 
 <script>
     $(document).ready(function() {
```
Comment (zhx828) [inline portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp]: Should we remove the ready event?

File: portal/src/main/webapp/js/lib/vc-session.js [hunk 2]
```diff
@@ -378,7 +378,7 @@ window.vcSession = window.vcSession ? window.vcSession : {};
       clickSave: function(_virtualStudy) {
         this.edit = false;
         if (_virtualStudy.studyName === '') {
-          _virtualStudy.studyName = 'My virtual study';
+            _virtualStudy.studyName = "Custom Cohort (" + new Date().toISOString().replace(/T/, ' ') + ")";
```
Comment (zhx828) [inline portal/src/main/webapp/js/lib/vc-session.js]: Should we make this a util in session-module then we can reuse it?

File: portal/src/main/webapp/js/src/cc-plots/cross-cancer-plotly-plots.js [hunk 3]
```diff
@@ -586,222 +584,7 @@ var ccPlots = (function (Plotly, _, $) {
 
 }(window.Plotly, window._, window.jQuery));
 
-var mutationStyle = (function() {  //Key and "typeName" are always identical
```
Comment (zhx828) [inline portal/src/main/webapp/js/src/cc-plots/cross-cancer-plotly-plots.js]: Basically moved to cc-plots folder without modifying any code, right?

File: portal/src/main/webapp/js/src/cc-plots/cross-cancer-plotly-plots.js [hunk 4]
```diff
@@ -487,80 +487,78 @@ var ccPlots = (function (Plotly, _, $) {
...
+                                result_str += _obj.sample_id + "\t" + _obj.study_name + "\t" + "RNA Seq V2" + "\t" + gene[0] + "\t" + "Not Sequenced" + "\t" + _obj.profile_data + "\n";
+                            }
+                        });
+                        return result_str;
+                    };
+                    cbio.download.clientSideDownload([get_tab_delimited_data()], "plots-data.txt");
+                });
+
+                // fetch data and init view
+                fetch_profile_data(Object.keys(_studySampleMap));
```
Comment (zhx828) [inline portal/src/main/webapp/js/src/cc-plots/cross-cancer-plotly-plots.js]: Just give Karthik a heads up, the only change in this block is `fetch_profile_data(Object.keys(_studySampleMap));` from `fetch_profile_data(_.pluck(_.pluck(window.studies.models, "attributes"), "studyId"));`

File: portal/src/main/webapp/js/src/cc-plots/cross-cancer-plotly-plots.js (hunk 3 above)
Comment (yichaoS) [inline portal/src/main/webapp/js/src/cc-plots/cross-cancer-plotly-plots.js]: yes, just separate this code into a different file. nothing changed.

File: portal/src/main/webapp/js/lib/vc-session.js (hunk 2 above)
Comment (yichaoS) [inline portal/src/main/webapp/js/lib/vc-session.js]: Yes. I'll do this when doing https://github.com/cBioPortal/cbioportal/pull/2286

File: portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp (hunk 1 above)
Comment (yichaoS) [inline portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp]: This file is not being used at all. I'll create another PR just to remote all the ghost files from the old "cross-cancer.do". This file is one of them.

File: portal/src/main/webapp/WEB-INF/jsp/query_form.jsp [hunk 5]
```diff
@@ -130,7 +130,7 @@
     window.is_virtual_cohort = '<%= isVirtualCohort %>';
     var _str = '<%= localCaseIds %>' === 'null'? '': '<%= localCaseIds %>'.trim();
     _str = _str.replace(/\+/g, '\n');
-    _str = _str.replace(/\|\|/g, '\t');
+    _str = _str.replace(/\|/g, '\t');
```
Comment (zhx828) [inline portal/src/main/webapp/WEB-INF/jsp/query_form.jsp]: Why we changed to this?

Comment (yichaoS) [inline portal/src/main/webapp/WEB-INF/jsp/query_form.jsp]: This is just a temporary/hacky solution. This will be gone.

**Pull request merged**